_partials/_devops-rest-api-get-started.md

```diff
@@ -0,0 +1,176 @@
+import RESTPrereqs from "versionContent/_partials/_prereqs-cloud-account-only.mdx";
+
+[$REST_LONG][rest-api-reference] is a comprehensive RESTful API you use to manage $CLOUD_LONG resources
+including VPCs, services, and read replicas.
+
+This page shows you how to set up secure authentication for the $REST_LONG and create your first service.
+
+## Prerequisites
+
+<RESTPrereqs />
+
+* Install [curl][curl].
+
+
+## Configure secure authentication
+
+$REST_LONG uses HTTP Basic Authentication with access keys and secret keys. All API requests must include
+proper authentication headers.
+
+<Procedure> 
+
+1. **Set up API credentials**
+
+    1. In $CONSOLE [copy your project ID][get-project-id] and store it securely using an environment variable:
+
+      ```bash
+      export TIGERDATA_PROJECT_ID="your-project-id"
+      ```
+
+    1. In $CONSOLE [create your client credentials][create-client-credentials] and store them securely using environment variables:
+
+       ```bash
+       export TIGERDATA_ACCESS_KEY="Public key"
+       export TIGERDATA_SECRET_KEY="Secret key"
+       ```
+
+1. **Configure the API endpoint**
+
+   Set the base URL in your environment:
+
+    ```bash
+    export API_BASE_URL="https://console.cloud.timescale.com/public/api/v1"
+    ```
+
+1. **Test your authenticated connection to $REST_LONG by listing services**
+
+    ```bash
+    curl -X GET "${API_BASE_URL}/projects/${TIGERDATA_PROJECT_ID}/services" \
+      -H "Authorization: Basic $(echo -n "${TIGERDATA_ACCESS_KEY}:${TIGERDATA_SECRET_KEY}" | base64)" \
```

Comment:
Fwiw, `curl` also has a `--user` flag that can be used to provide these values without having to base64 encode them yourself.